walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -37,8 +41,34 @@
       return new ArrayList<TimeRange>();
     }
 
-    return traverseThroughSchedule(request, eventList);
+    Collection<TimeRange> result = traverseThroughSchedule(request, eventList);
+    if (checkIfAvailability(result, eventList2)) {
```

Comment:
This checkIfAvailability function is a bit complicated. Why not run `traverseThroughSchedule` of your combined group first, then if result.isEmpty() is true, you run it on just the required attendees.

Reply:
10/10